modules/ELT/pages/getting-started-with-airbyte.adoc

```diff
@@ -0,0 +1,197 @@
+= Airbyte with Teradata Vantage
+:experimental:
+:page-author: Krutik Pathak
+:page-email: krutik.pathak@teradata.com
+:page-revdate: June 1st, 2023
+:description: Use Airbyte with Teradata Vantage.
+:keywords: airbyte, data warehouses, compute storage separation, teradata, vantage, cloud data platform, object storage, data transfer, data extraction, enterprise analytics, elt.
+:tabs:
+:dir: getting-started-with-airbyte
+
+== Overview
+
+This tutorial showcases how to use Airbyte (an Open-source Extract Load Transform Tool) with Teradata Vantage. We work with a very simple end-to-end setup to load the employees’ data from Google Sheets to Teradata Vantage using Airbyte.
+
+
+image::{dir}/sample_employees_payrate_google_sheets.png[Sample Employees Payrate Google Sheets, width=75%]
+
+* Source: Google Sheets
+* Destination: Teradata Vantage
+
+== Prerequsites
+* Access to Teradata Vantage Instance. As this will be our destination. You will need a database host, username, and password for Airbyte’s configuration.
++
+include::ROOT:partial$vantage_clearscape_analytics.adoc[]
+
+* docker-compose to run the link:https://github.com/airbytehq/airbyte[Airbyte open-source] locally. docker-compose comes with Docker Desktop. Please refer to link:https://docs.docker.com/compose/install/[docker docs] for more details. 
+
+* link:https://support.google.com/googleapi/answer/6158841?hl=en[Google Cloud Platform API enabled for your personal or organizational account]. You’ll need to authenticate your Google account via OAuth or via Service Account Key Authenticator.
```

Comment:
I think Google APIs is different from GCP API?

Reply:
@adamtwo yes they are different. Here it is asking to authenticate the google account either through Service Account Key Auth or OAuth. For the same it is asking to enable GCP API.